crates/hotshot/src/tasks/task_state.rs

```diff
@@ -207,3 +207,19 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> CreateTaskState<TYPES, I>
         }
     }
 }
+
+#[async_trait]
+impl<TYPES: NodeType, I: NodeImplementation<TYPES>> CreateTaskState<TYPES, I>
+    for QuorumVoteTaskState<TYPES, I>
+{
+    async fn create_from(handle: &SystemContextHandle<TYPES, I>) -> QuorumVoteTaskState<TYPES, I> {
+        QuorumVoteTaskState {
+            latest_voted_view: handle.get_cur_view().await,
+            vote_dependencies: HashMap::new(),
+            quorum_network: handle.hotshot.networks.quorum_network.clone(),
+            committee_network: handle.hotshot.networks.da_network.clone(),
```

Comment:
Just wondering: `da_network` and `committee_network` mean the same thing, right?

Should we just pick one of the names and replace the other? (no preference for which one, and we can just do this later too if it's not quick/you just want to merge this in)

Reply:
I didn't do this renaming (but did the other one for the event name) since we have a lot of mixed-use of "da" and "committee". I'd like to do a renaming for all in a separate PR.